test/mps.jl

```diff
@@ -6,7 +6,7 @@ using Test
 Random.seed!(1234)
 
 include("util.jl")
-include(joinpath(pkgdir(ITensors), "examples", "gate_evolution", "qubit.jl"))
+include(joinpath(pkgdir(ITensors), "examples", "gate_evolution"))
```

Comment:
Probably this entire line isn't needed anymore (this tries to `include` a directory, which errors).

Reply:
Yes, thanks. Just fixed that one. Also now just fixing up a few unit tests that use some of the more esoteric Qubit operators like the "randn" one, which I think I didn't bring over because I didn't know it was being used anywhere.

Reply:
It was just used in testing out the gate evolution functionality, it's not necessary.

A better one to add would be a "RandomUnitary" gate like we have in PastaQ (https://github.com/GTorlai/PastaQ.jl/blob/master/src/circuits/gates.jl#L305-L313) which makes a Haar random unitary of any number of indices. That can be left for a future PR though.

Reply:
Just what I needed to know - thanks!